apps/src/javalab/PlaygroundVisualizationColumn.jsx

```diff
@@ -79,7 +82,8 @@ const styles = {
   playground: {
     backgroundColor: 'white',
     width: 800,
-    height: 800
+    height: 800,
+    overflow: 'hidden'
```

Comment:
🎉

Reply:
Also FYI I didn't remove the clip-path stuff yet, planning on doing that as an immediate follow-up to make sure it works as expected.